packages/rich-text/src/plugins/shared/FetchingWrappedAssetCard.tsx

```diff
@@ -36,6 +34,10 @@ const InternalAssetCard = React.memo((props: InternalAssetCardProps) => {
     );
   }
 
+  if (!props.asset) {
```

Comment:
Shouldn't we explicitly check for `loading`?

Reply:
would need some more refactorings, can propose that in a follow up PR 👍

Reply:
@z0al what do you think: https://github.com/contentful/field-editors/pull/1643